src/st/update_stage_entities.c

```diff
@@ -1,3 +1,7 @@
+#include <stage.h>
+
+u16 UNK_Invincibility0[];
```

Comment:
Actually we seem to have this declared multiple times? See update.c. Also I thought these needed to be extern but apparently not? That's a mistake I made on an earlier PR.